common/src/com/thoughtworks/go/server/websocket/Message.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.server.websocket;
+
+import com.thoughtworks.go.util.SystemEnvironment;
+import org.eclipse.jetty.websocket.api.WebSocketPolicy;
+
+import java.io.*;
+
+public class Message {
+
+    public static void setupPolicy(WebSocketPolicy policy, SystemEnvironment environment) {
+        policy.setMaxTextMessageBufferSize(environment.getWebsocketMaxMessageSize());
+        policy.setMaxTextMessageSize(environment.getWebsocketMaxMessageSize());
+        policy.setMaxBinaryMessageSize(environment.getWebsocketMaxMessageSize());
+        policy.setMaxBinaryMessageBufferSize(environment.getWebsocketMaxMessageSize());
+    }
+
+    public static byte[] encode(Message msg) {
+        String encode = JsonMessage.encode(msg);
```

Comment:
If we're serializing/de-serializing as JSON, why do we encode things as bytes instead of just sending strings back and forth?